examples/gnrc_lorawan/main.c

```diff
@@ -0,0 +1,126 @@
+/*
+ * Copyright (C) 2019 HAW Hamburg
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup     tests
+ * @{
+ * @file
+ * @brief       Test application for GNRC LoRaWAN
+ *
+ * @author      José Ignacio Alamos <jose.alamos@haw-hamburg.de>
+ * @}
+ */
+
+#include <errno.h>
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+
+#include "thread.h"
+#include "xtimer.h"
+#include "shell.h"
+#include "shell_commands.h"
+
+#include "board.h"
+
+#include "net/gnrc/netapi.h"
+#include "net/gnrc/netif.h"
+
+#include "net/gnrc/pktbuf.h"
+#include "net/gnrc/netreg.h"
+
+#define LORAWAN_PORT (2U)
+
+static void _netreg_cb(uint16_t cmd, gnrc_pktsnip_t *pkt, void *ctx)
+{
+    (void) cmd;
+    (void) ctx;
+    puts("Received data:");
+    for(unsigned  i=0; i<pkt->size;i++)
+    {
+        printf("%02x ", ((uint8_t*) pkt->data)[i]);
+    }
+    printf("\n");
+    gnrc_pktbuf_release(pkt);
+}
+
+gnrc_netreg_entry_cbd_t _cbd = {
+    .cb = _netreg_cb,
+    .ctx = NULL
+};
+
+static gnrc_netreg_entry_t _entry = GNRC_NETREG_ENTRY_INIT_CB(LORAWAN_PORT, &_cbd);
```

Comment:
Why not use straight-up `gnrc_pktdump` for that? It basically copies its functionallity (except for only printing the payload).

Reply:
yes, I think we can use that

Reply:
addressed in https://github.com/RIOT-OS/RIOT/pull/11022/commits/ebe141a79582016423e4b86ba13d678d1b11c82d and https://github.com/RIOT-OS/RIOT/pull/11022/commits/968d64837885c78d6ba0a9921d4263e674462617